rifServices/src/main/java/org/sahsu/rif/services/graphics/RIFMapsParameters.java

```diff
@@ -81,6 +81,13 @@
  */
 	
 public class RIFMapsParameters {
+
+	
+	enum StudyType {
+		RISK_ANALYSIS, 
+		DISEASE_MAPPING};
```

Comment:
There's already a `StudyType` enum: `org.sahsu.rif.services.concepts.StudyType`. Better to use that one than inttroduce a new one.

Reply:
Fixed